DOC-3809 Updated key concepts states (#560)
corda/corda-docs-portal: content/en/platform/corda/4.8/open-source/key-concepts-states.md

## Summary

* States represent facts on the **[ledger](key-concepts-ledger.md)**.

> * Facts evolve on the ledger by participants creating new states and marking outdated states as historic.

Reviewer: ```suggestion
* Facts evolve on the ledger when participants create new states and mark outdated states as historic.
```

* Each node has a vault where it stores the states it shares with other nodes.

## Video

{{% vimeo 213812054 %}}

## Overview

A state is an immutable object representing a fact known by one or more nodes at a specific point in time.

> States can contain arbitrary data, allowing them to represent facts of any kind. For example, stocks, bonds, loans, KYC data,
> or identity information.

Reviewer: ```suggestion
You can use states to represent any type of data, and any kind of fact. For example, a financial instrument, Know Your Customer (KYC) data, or identity information.
```

> This state represents an IOU—an agreement that Alice owes Bob £10:

Reviewer: 💖 THANK YOU

{{< figure alt="state" width=80% zoom="/en/images/state.png" >}}

> As well as information about the fact itself, the state also contains a reference to the

Reviewer: ```suggestion
In addition to information about the fact, the state contains a reference to the
```

**[contract](key-concepts-contracts.md)**. Contracts govern the evolution of states.

## State sequences

> As states are immutable, they cannot be modified. Therefore, Corda uses **state sequences** to track the evolution of facts.

Reviewer: ```suggestion
States are immutable: you can't change them. Corda uses **state sequences** to track the evolution of facts.
```
